migration/sql-files/005-add-search-index.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS fields_title ON work_items USING GIN (to_tsvector('english', fields->>'system.title'));
```

Comment:
Thanks @aslakknutsen 
I've imported the dump, and now `select count(1) from work_items;` gets me 132119 items.

I see the indexes are being used.

In my pgadmin console ( the output of explain analyze is not copy-able :| )
Here's the output line about index :

```
Index Cond: 
(((setweight(to_tsvector('english'::regconfig, COALESCE((fields ->> 'system.title'::text), ''::text)), 'B'::"char")
 || setweight(to_tsvector('english'::regconfig, COALESCE((fields ->> 'system.description'::text), ''::text)), 'C'::"char"))
 || setweight(to_tsvector('english'::regconfig, (id)::text), 'A'::"char")) 
@@ '''115742'':* | ''demo.almighty.io/detail/115742'':* & ''demo.almighty.io'':* & ''/detail/115742'':*'::tsquery)
```

Execution time is a decent 0.139ms